client/hudwidget.cpp

```diff
@@ -259,24 +259,52 @@ hud_text::hud_text(const QString &s, int time_secs, QWidget *parent)
 void hud_text::show_me()
 {
   show();
-  center_me();
+  position_me(5);
 }
 
 /**
-   Moves to top center parent widget and sets size new size
+   Moves to position in parent widget and sets size new size
  */
-void hud_text::center_me()
+void hud_text::position_me(int corner)
 {
+  /*
+   *  position pattern
+   *
+   *  |1 ... 2|
+   *  |...5...|
+   *  |.......|
+   *  |3 ... 4|
+   */
```

Comment:
Except for the center, this looks a lot like [`Qt::Corner`](https://doc.qt.io/qt-5/qt.html#Corner-enum). It's in any case better to use an enum than magic numbers.